src/zenml/integrations/aws/service_connectors/aws_service_connector.py

```diff
@@ -1371,6 +1372,21 @@ def _configure_local_client(
                     "aws_session_token"
                 ] = credentials.token
 
+            aws_credentials_path = os.path.join(
+                users_home, ".aws", "credentials"
+            )
+
+            # Create the file as well as the parent dir if needed.
+            dirname = os.path.split(aws_credentials_path)[0]
```

Comment:
I don't think this would be an issue since that code is supposed to run inside the system which would rely on `os` without the need for `fileio` one other thing `fileio.open()` function is limited compared to `os.open()`

Reply:
ok! then ignore